src/main/java/com/atlauncher/viewmodel/impl/AboutTabViewModel.java

```diff
@@ -35,66 +35,18 @@
  * 13 / 06 / 2022
  */
 public class AboutTabViewModel implements IAboutTabViewModel {
-    /**
-     * Produced via "git shortlog -s -n --all --no-merges" then some edits
-     * <p>
-     * Use the following pattern to retrieve icons
-     * "https://avatars.githubusercontent.com/USERNAME"
-     */
-    @SuppressWarnings("JavadocLinkAsPlainText")
-    private static final String[] AUTHORS_ARRAY = {
-        "Ryan Dowling",
-        "RyanTheAllmighty",
-        "doomsdayrs",
-        "Asyncronous",
-        "PORTB",
-        "JakeJMattson",
-        "Jamie (Lexteam)",
-        "Ryan",
-        "s0cks",
-        "Jamie Mansfield",
-        "flaw600",
-        "Leah",
-        "Alan Jenkins",
-        "dgelessus",
-        "Kihira",
-        "Harald Krämer",
-        "James Ross",
-        "iarspider",
-        "xz-dev",
-        "Mysticpasta1",
-        "Torsten Walluhn",
-        "modmuss50",
-        "Andrew Thurman",
-        "Cassandra Caina",
-        "Jamie (Lexware)",
-        "Jowsey",
-        "Shegorath123",
-        "Tazz",
-        "notfood",
-        "Dallas Epperson",
-        "Emma Waffle",
-        "Hossam Mohsen",
-        "JBMagination",
-        "Jamie",
-        "Laceh",
-        "Mihail Yaremenko",
-        "Sasha Sorokin",
-        "TecCheck",
-        "Trejkaz",
-        "mac",
-    };
 
     private String info = null;
 
     @Nonnull
     @Override
-    public List<Author> getAuthors() {
-        // Since the source is git, we can just map it
-        return Arrays
-            .stream(AUTHORS_ARRAY)
-            .map(author -> new Author(author, "https://avatars.githubusercontent.com/" + author))
-            .collect(Collectors.toList());
+    public List<Contributor> getContributors() {
+        GetLauncherContributorsQuery.Data response = GraphqlClient
+                            .callAndWait(new GetLauncherContributorsQuery(), 1, TimeUnit.DAYS);
+
+        return response.about().contributors().stream().map(contributor -> {
+            return new Contributor(contributor.name(), contributor.url(), contributor.avatarUrl());
+        }).collect(Collectors.toList());
     }
```

Comment:
Consider the following.

AboutTab destroys the UI whenever the hierarchy is changed.

This function `getContributors` is called whenever the UI is being created.
This is a long standing operation occurring on the UI lifecycle, every time the view is recreated.

Previously, this function worked fine because the data was loaded into memory as a static field.

Now, instead we may consider a more reactive setup. Which I shall make a commit for shortly.

Essentially, the function will be converted to return an RxObservable, with an initial value of an empty array. This will let the UI render along side as the data is queried.

Reply:
Addendum, There is a built in cache that helps subsequent calls.

This is still not optimal for a first time load, especially on a slower internet connection.

Reply:
Right, I need to get into the mindset of utilising RxJava more

Reply:
Completed with [72b452c](https://github.com/ATLauncher/ATLauncher/pull/903/commits/72b452cdc6ab3d866260cc580f87e85dc0a327ef) and [530d08e](https://github.com/ATLauncher/ATLauncher/pull/903/commits/530d08e6db4f27b80ffd3d73f02b057819bed7a4)

Reply:
Close when acknowledged